content/ending-frozen-applications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ending Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Ending Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to end hanging or frozen processes. The `gnome-system-monitor` provides a GUI process management application with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
+
+>Note: This article details several methods to end processes for a given scenario, but is not exhaustive or guaranteed to provide the most appropriate method for every scenario. Users should also keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.
```

Comment:
> Note:

Can this `Note:` label be omitted? Is it unclear that this block of text is a note? As it relates to the second sentence, should the label say `Caution:` or `Warning:` instead?

> This article details several methods to end processes for a given scenario, but is not exhaustive or guaranteed to provide the most appropriate method for every scenario.

I'm not sure it's necessary or desirable to point out that the article doesn't cover all scenarios. That information isn't specific and may decrease confidence in the article.

> Users should also keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.

This is a perfectly fair warning. Maybe this line could be reduced to just:

> Caution: Keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.

Reply:
Good point, I'll apply suggested changes. I added the first part because there's always the "why would you do it _that_ way?" Linux guru, but on second glance don't think it really adds much to the article.